source/Cosmos.Core/PIC.cs

```diff
@@ -95,8 +95,8 @@ private void Remap(byte masterStart, byte masterMask, byte slaveStart, byte slav
             // set masks:
             Master.Data.Byte = masterMask;
             IOPort.Wait();
-            Slave.Data.Byte = slaveMask;
-            IOPort.Wait();
+            //Slave.Data.Byte = slaveMask;
```

Comment:
Not sure if this is the correct fix. I think that the problem is the mask (passed on line 47). I'm not sure why the mask is set to `0xEF`, but probably there's some readon. Try replacing `0xEF` with `0xEB`.